api/workAllocation/util.ts

```diff
@@ -520,14 +520,20 @@ export function filterMyAccessRoleAssignments(roleAssignmentList: RoleAssignment
 
 export async function getMyAccessMappedCaseList(roleAssignmentList: RoleAssignment[], req: EnhancedRequest)
   : Promise<RoleCaseData[]> {
+
+  const newRoleAssignment = getAccessGrantedRoleAssignments(roleAssignmentList);
+
   const specificRoleAssignments = filterMyAccessRoleAssignments(roleAssignmentList);
 
   const cases = await getCaseIdListFromRoles(specificRoleAssignments, req);
+  return mapCasesFromData(cases, specificRoleAssignments, newRoleAssignment);
+}
 
-  return mapCasesFromData(cases, specificRoleAssignments);
+export function getAccessGrantedRoleAssignments(roleAssignmentList: RoleAssignment[]): RoleAssignment[] {
+  return roleAssignmentList.filter(r => r.roleName === 'specific-access-granted')
 }
 
-export function constructElasticSearchQuery(caseIds: any[], page: number, size: number): ElasticSearchQuery [] {
+export function constructElasticSearchQuery(caseIds: any[], page: number, size: number): ElasticSearchQuery[] {
```

Comment:
caseIds is a list of string?

Reply:
The change is in the white space. Changing this has to be tested and not releated to this JIRA